opensciencegrid/gwms-factory/Dockerfile

```diff
@@ -21,3 +21,5 @@ RUN    yum -y install \
     && rm -rf /var/cache/yum/*
 
 ADD etc /etc
+ADD usr /usr
+ADD http://mmascher.web.cern.ch/mmascher/OSG_autoconf /tmp
```

Comment:
This seems a little brittle, no? We should be throwing versions of scripts that we can ostensibly track the provenance of (i.e., version number, git revision, etc)

If you want to do development on the script, you can just shell into the running container and curl your script.

Reply:
Mmk, sounds good